### DISCUSSION TITLE
Get max and min of attribute from qdrant metadata

### DISCUSSION THREAD

**Discussion by Thiru-GVT** [qdrant/qdrant#3198] [Q&A]
Hello, I have metadata in my vectors and one of it is the date.

I understand theres no native date time format, so int storage has to suffice. However i dont see a way of finding the maximum or min from the api?

**Comment by joein** [comment]
Hi @Thiru-GVT 

Unfortunately, there is no API to retrieve max or min value of a payload field at the moment.

If you know the scale of your variable, you can make scroll requests with [range](https://qdrant.tech/documentation/concepts/filtering/#range) filters, e.g. `gt=<value>`, or if you don't - you would need to scroll through the whole collection :(

We are going to have a `cardinality API` in one of the next releases, which will allow to fetch all unique values of a payload field. Might help to solve your task as well (still not the most efficient way to do it though)

**Reply by NaxYo** [reply]
Hi @joein , any updates on this?

> We are going to have a `cardinality API` in one of the next releases, which will allow to fetch all unique values of a payload field

I'm working on a use case where we need to _lookup what's there_, and that cardinality API you mention would most likely do the trick.

**Reply by timvisee** [reply]
No updates. We still have this planned, but have no expected release date yet.
